### ISSUE TITLE
The assets should not be in this repo

### ISSUE THREAD

**Issue by jdetter** [UW-UPL/UPL-MMORPG#3]
The assets should be downloaded on launch, probably from a public folder for now. Adding more assets will bloat the repo a ton.

**Comment by MikeCook9994** [comment]
You could use svn instead.

**Comment by jdetter** [comment]
Meh, it will take me longer to move everything over to svn then it would be for me to literally write code to download and untar a tarball on startup when the assets folder is empty. I know the git features better, and I'm assuming everyone else wants it on github =D

**Comment by apiedev** [comment]
With a game like this, couldn't we just do asset downloading upon entering each map? Like how most Gmod servers function, if you don't have the assets, it just downloads them quick before you enter the server/map/room.

**Comment by katamaritaco** [comment]
Tbh until project grows larger consider using dropbox for assets

Get Outlook for Androidhttps://aka.ms/ghei36

On Wed, Aug 10, 2016 at 1:22 AM -0500, "Alec Pierce" <notifications@github.com<mailto:notifications@github.com>> wrote:

With a game like this, couldn't we just do asset downloading upon entering each map? Like how most Gmod servers function, if you don't have the assets, it just downloads them quick before you enter the server/map/room.

##

**Comment by jdetter** [comment]
Yeah we could do something like dropbox ^^ For now I say we just keep the assets in the repo until we have like a stable 'real' game. By then we will ready for something like dropbox or possibly hosting from a public folder or something like that.

**Comment by spwilson2** [comment]
[git lfs](https://git-lfs.github.com/) is an option too. Just would add a dependency for development.

**Comment by spwilson2** [comment]
And then once you hit a release milestone: [https://help.github.com/articles/distributing-large-binaries/](https://help.github.com/articles/distributing-large-binaries/)

**Comment by katamaritaco** [comment]
I've heard that git lfs is still not the best for games but that could just be coming from jaded game devs. As somebody who isn't working on this, I think it'd be great to try it out ah3h

Get Outlook for Androidhttps://aka.ms/ghei36

On Tue, Aug 16, 2016 at 8:20 AM -0500, "Sean Wilson" <notifications@github.com<mailto:notifications@github.com>> wrote:

git lfs is an option too. Just would add a dependency for development.

##
